sqlmodel/main.py

```diff
@@ -11,6 +11,7 @@
 from typing import (
     AbstractSet,
     Any,
+    Annotated, 
```

Comment:
Annotated was introduced in Python 3.9. Need to check python version and import it from typing_extensions if necessary

Reply:
Ok, use typing_extensions to import `Annotated`.